Adds in an "Advanced Who"

Cameron653 opened the pull request:
Adds in an "Advanced who" which lets normal players see if someone is observing/in the lobby/playing.
Same as it was in the old code, except this one is listed as an "Advanced Who" as it gives players more information, as opposed to the normal who, which only tells you who's on the server.

SpadesNeil commented:
I feel like I used to have a reason for not having this, but for the life of me, I can't remember what it was.

If it's just like the old version I see no reason not to add it because I obviously approved that.

Cameron653 commented:
The reason it (before I changed it) was that it showed AFK players to
normal players. I removed it and it was approved. Of course, it was with
the normal who, but I didn't want to touch the pre existing who verb in the
new code.
